tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -397,14 +397,14 @@ module rec Compiler =
 
     let withLangVersionPreview (cUnit: CompilationUnit) : CompilationUnit =
         withOptionsHelper [ "--langversion:preview" ] "withLangVersionPreview is only supported on F#" cUnit
-        
+
```

Comment:
Changes to this file appear to have snuck in from my other branch, sorry, but I claim they are good changes.